apps/main/src/lend/components/MarketInformationComp.tsx

```diff
@@ -33,22 +40,71 @@ export const MarketInformationComp = ({
   page = 'manage',
 }: MarketInformationCompProps) => {
   const { rChainId, rOwmId, market } = pageProps
+  const collateralTokenAddress = market?.collateral_token.address
+  const borrowedTokenAddress = market?.borrowed_token.address
+  const api = getLib('llamaApi')
   const theme = useTheme()
+  const [releaseChannel] = useReleaseChannel()
+  const isBeta = releaseChannel === ReleaseChannel.Beta
```

Comment:
Let's keep the features together in [`useFeatureFlags`](https://github.com/curvefi/curve-frontend/blob/4f887c9d28dbb373e6f4ac1e6ab426c263637958/packages/curve-ui-kit/src/hooks/useFeatureFlags.ts#L9) please

```suggestion
  const shouldUseNewChart = useNewBandsChart()
```

Reply:
Ah yes, I forgot to refactor to this after it got introduced 🤝